### PR TITLE
MTM-52611 add alarms redesign announcement

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52611
+version: 1020.0.0
+---
+In a future version, we are introducing redesigned alarms with both visual and functional enhancements.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:17]:
```suggestion
In a future version, we will introduce redesigned alarms with both visual and functional enhancements.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**Visual and UX Enhancements**:
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:19]:
```suggestion
**Visual and UX enhancements**:
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+* We have redesigned the way alarms are displayed in the "Alarms" view and widgets.
+
+**Functional Improvements**:
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:23]:
```suggestion
**Functional improvements**:
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+* The "Alarms" view now shows only a single list of alarms, where filters can be applied.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:25]:
```suggestion
* The "Alarms" view now only shows one single list of alarms, in which filters can be applied.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+* We have introduced a details section, which contains all information related to the alarm.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:26]:
```suggestion
* A details section has been introduced which contains all information related to the alarm.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+* The functionalities of the "Alarm list", "All critical alarms", and "Recent alarms" widgets have been merged into a single, comprehensive widget called "Alarm list".
+* The Alarm list widget has been migrated to Angular.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:28]:
```suggestion
* The "Alarm list" widget has been migrated to Angular.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+* Real-time functionality has been replaced with Auto refresh.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:29]:
```suggestion
* The real-time functionality has been replaced with an auto-refresh functionality.
```

**Review by rahultsag — CHANGES_REQUESTED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52611
+version: 1020.0.0
+---
+In the upcoming version, we will introduce redesigned alarms with both visual and functional enhancements.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:17]:
```suggestion
In the upcoming version, a redesign of the alarms feature will be introduced with both visual and functional enhancements.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52611
+version: 1020.0.0
+---
+In the upcoming version, a redesign of the alarms feature will be introduced with both visual and functional enhancements.
+
+**Visual and UX enhancements**:
+
+* We have redesigned the way alarms are displayed in the "Alarms" view and widgets.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md:21]:
```suggestion
* The way alarms are displayed in the "Alarms" view and widgets has been redesigned.
```

**Review by BeateRixen — APPROVED** [review]

**Review by rahultsag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@DawidSAG Can be merged from my side.

**Pull request merged**